sherpa-onnx/c-api/c-api.h

```diff
@@ -54,30 +54,37 @@ extern "C" {
 /// from there.
 SHERPA_ONNX_API typedef struct SherpaOnnxOnlineTransducerModelConfig {
   const char *encoder;
+  const char *encoder_buf_begin, *encoder_buf_end;  // if non-null, loading the encoder from the buffer in prioriy
   const char *decoder;
+  const char *decoder_buf_begin, *decoder_buf_end;  // if non-null, loading the decoder from the buffer in prioriy
   const char *joiner;
+  const char *joiner_buf_begin, *joiner_buf_end;  // if non-null, loading the joiner from the buffer in prioriy
```

Comment:
Please add new fields to the end of existing fields.

Please don't change the order of exsiting fields.

Reply:
I suggest that you use
```
const char *encoder_buf_begin;
int32_t encoder_buf_len;
```